Article/Article.js

```diff
@@ -3,24 +3,27 @@
 class Article {
   constructor(element) {
     // assign this.element to the passed in article element
-    this.element;
+    this.element = element;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.element.querySelector(".expandButton");
+    // console.log(this.expandButton);
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerHTML = "expand";
     // Set a click handler on the expandButton reference (or article element), calling the expandArticle method.
+    this.expandButton.addEventListener('click', this.expandArticle)
     
   }
 
-  expandArticle() {
+  expandArticle(event) {
     // Using our reference to the article element, add or remove a class to expand or hide the article.
+    event.currentTarget.parentNode.classList.toggle('article-open');
```

Comment:
That's a very long chain. It is apparent that you did your research.

Reply:
Thanks! I feel like I am getting a hang of chaining. It seems to make things a lot cleaner once I understand it.